Bottom: 40c997c7a416141f18f67a6a0e3f46260212efab
Top:    9b32809ce290ae9dfabc8c0d24c3fc647aeb25c3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 02:28:47 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/crosshair.c b/src/crosshair.c
index ac22277..aaa46ae 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -694,8 +694,10 @@ CrosshairOn (bool BlockToo)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -708,8 +710,10 @@ CrosshairOff (bool BlockToo)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -738,7 +742,7 @@ HideCrosshair (bool BlockToo)
   CrosshairStack[CrosshairStackLocation] = Crosshair.On;
   CrosshairStackLocation++;
 
-//  CrosshairOff (BlockToo);
+  CrosshairOff (BlockToo);
 }
 
 /* ---------------------------------------------------------------------------
@@ -758,11 +762,11 @@ RestoreCrosshair (bool BlockToo)
 
   if (CrosshairStack[CrosshairStackLocation])
     {
-//      CrosshairOn (BlockToo);
+      CrosshairOn (BlockToo);
     }
   else
     {
-//      CrosshairOff (BlockToo);
+      CrosshairOff (BlockToo);
     }
 }
